src/topp/IDMapper.cpp

```diff
@@ -92,7 +94,12 @@ using namespace std;
 
     <B>Annotation of consensus maps (consensusXML input):</B>\n
     Peptide positions are always matched against centroid positions. By default, the consensus centroids are used. However, if @p consensus:use_subelements is set, the centroids of sub-features are considered instead.
-    In this case, a peptide identification is mapped to a consensus feature if any of its sub-features matches.
+    In this case, a peptide identification is mapped to a consensus feature if any of its sub-features matches. In the case of consensusXMLs from TMT data, generated by
+    IsobaricAnalyzer, this tool will use the spectrum ID attached to the feature to retrieve an identification from the idXML.
+    
+    <B>Annotation with mzML (both feature and consensusXML input):</B>\n
+    Instead of/In addition to annotating with putatively identified spectra (i.e., PSMs) from idXML, you can use the original spectra in an mzML to attach
+    all potential MS2 spectra to a (Consensus)Feature. See the @p spectra:in option.
```

Comment:
it may not be entirely clear what exactly is annotated (the spectrum index/reference for the unidentified spectrum?) and why this is useful?!

Reply:
Regarding usefulness, you have to do a git blame and find the author and ask them.

Reply:
use case: find all MS2 spectra of a quantified peptide to e.g., generate spectral libraries

Reply:
> use case: find all MS2 spectra of a quantified peptide to e.g., generate spectral libraries

then the docs need to be reworked. They say `annotate unidentified spectra`. So you cannot possibly make a speclib out of it.

Reply:
Sure you can ;) Speclib of unknown metabolites